azure-pipelines.yml

```diff
@@ -213,15 +198,11 @@ stages:
                   $signAssemblies = [System.Convert]::ToBoolean("$(IS_RELEASE_BRANCH)") # the variable is a string, we need a boolean
                   Package-NetFrameworkScanner -SignAssemblies $signAssemblies
                   Package-NetScanner -SignAssemblies $signAssemblies
-                pwsh: true
 
-            - task: NuGetCommand@2
+            - powershell: |
+                nuget help | Select-String "NuGet Version"
+                nuget pack 'nuspec\netcoreglobaltool\dotnet-sonarscanner.nuspec' -NonInteractive -OutputDirectory build -Verbosity Detailed
```

Comment:
`NuGetCommand@2` uses an older version of `nuget.exe`

Reply:
See [this comment](https://github.com/SonarSource/sonar-scanner-msbuild/pull/1933#discussion_r1639843424).
And [this documentation](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/nuget-command-v2?view=azure-pipelines#remarks).
It was done to force the use of a newer Nuget version than 4.1.0 (which is the default for this task and can't be configured through parameters).